kbdcfg.lua

```diff
@@ -130,6 +134,7 @@ local function factory(args)
     kbdcfg.current               = args.current or kbdcfg.default_layout_index
     kbdcfg.menu                  = nil
     kbdcfg.type                  = args.type or "tui"
+    kbdcfg.no_tui_margin         = args.no_tui_margin or false
```

Comment:
I would prefer if we will have option `tui_space_symbol` or something like that. And by default it should be equal to " ". In this case, user will be able to customize it. This option will be more flexible as `no_tui_margin`.